cmd/kured/main.go

```diff
@@ -103,6 +107,13 @@ const (
 	KuredMostRecentRebootNeededAnnotation string = "weave.works/kured-most-recent-reboot-needed"
 	// EnvPrefix The environment variable prefix of all environment variables bound to our command line flags.
 	EnvPrefix = "KURED"
+
+	// MethodCommand is used as "--reboot-method" value when rebooting with the configured "--reboot-command"
+	MethodCommand = "command"
+	// MethodSignal is used as "--reboot-method" value when rebooting with a SIGRTMIN+5 signal.
+	MethodSignal = "signal"
+
+	sigTrminPlus5 = 34 + 5
```

Comment:
Is this meant to be sigRtminPlus5  ?

Reply:
Yes, I fix the typo.

Reply:
Looks like the typo is still there? But otherwise this should be finally good to go!

Reply:
Sorry, I didn't catch that. Let's fix the typo. If we're going to use camel case I'd vote for:

- `sigRTMin`

Maybe:

```go
const (
...
    sigRTMin = 34
    defaultSignal = sigRTMin + 5
...
)
```

And we should probably change the `MethodSignal` comment so that it doesn't imply that SIGRTMIN+5 is static (it's just a default, but the user can use whatever number within the min-max range that their OS allows). Maybe:

```
// MethodSignal is used as "--reboot-method" value when rebooting according to a signal in the allowed range between SIGRTMIN and SIGRTMAX as defined on the OS platform running kured
```